src/modules/colorPicker/ColorPickerUI/Keyboard/KeyboardMonitor.cs

```diff
@@ -56,6 +58,14 @@ private void ActivationShortcut_PropertyChanged(object sender, System.ComponentM
         private void Hook_KeyboardPressed(object sender, GlobalKeyboardHookEventArgs e)
         {
             var virtualCode = e.KeyboardData.VirtualCode;
+            // ESC pressed
+            if(virtualCode == 27)
```

Comment:
nit:   This would be more readable as if(virtualCode == KeyInterop.VirtualKeyFromKey(Key.Escape))

Reply:
done